src/page-elements/search-page.tsx

```diff
@@ -29,6 +30,9 @@ interface State {
 	corpora: readonly Corpus[];
 	unavailableCorpora: ReadonlySet<Corpus>;
 
+	betaResults: boolean;
+	refreshSearch: boolean; //set to true after pressing the showExperimental button to refresh the result list results
```

Comment:
This is unnecessary. The value is never changed, so you can remove this from the state.